games-action/openclonk-7.0

```diff
@@ -0,0 +1,118 @@
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	openclonk$secondaryArchSuffix = $portVersion
+	app:OpenClonk$secondaryArchSuffix = $portVersion
+	application$secondaryArchSuffix = $portVersion
+	bundled$secondaryArchSuffix = $portVersion
+	mimehandler$secondaryArchSuffix = $portVersion
+	openclonk(x86-64)$secondaryArchSuffix = $portVersion
+        "
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg_turbo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat-1.0k$secondaryArchSuffix
+	lib:libc$secondaryArchSuffix
+	lib:libcairo-gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgcc_s$secondaryArchSuffix
+	lib:libgdk_pixbuf-2.0$secondaryArchSuffix
+	lib:libgio-2.0$secondaryArchSuffix
+	lib:libglib-2.0$secondaryArchSuffix
+	lib:libgobject-2.0$secondaryArchSuffix
+	lib:libgthread-2.0$secondaryArchSuffix
+	lib:libgtk-3$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libm$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango-1.0$secondaryArchSuffix
+	lib:libpangocairo-1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpthread$secondaryArchSuffix
+	lib:librt$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:zlib$secondaryArchSuffix
+	"
+BUILD_REQUIRES"
+	haiku_devel$secondaryArchSuffix
+	devel:g++$secondaryArchSuffix
+        devel:zlib$secondaryArchSuffix
+        devel:libpng$secondaryArchSuffix
+        devel:libjpeg_turbo$secondaryArchSuffix
+        devel:libfreetype$secondaryArchSuffix
+        devel:libogg$secondaryArchSuffix
+        devel:libvorbis$secondaryArchSuffix
+        devel:libxrandr$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLEW$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libault$secondaryArchSuffix
+	devel:libat-1.0k$secondaryArchSuffix
+	devel:libc$secondaryArchSuffix
+	devel:libcairo-gobject$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libgcc_s$secondaryArchSuffix
+	devel:libgdk_pixbuf-2.0$secondaryArchSuffix
+	devel:libgio-2.0$secondaryArchSuffix
+	devel:libglib-2.0$secondaryArchSuffix
+	devel:libgobject-2.0$secondaryArchSuffix
+	devel:libgthread-2.0$secondaryArchSuffix
+	devel:libgtk-3$secondaryArchSuffix
+	devel:libixml$secondaryArchSuffix
+	devel:libm$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libpango-1.0$secondaryArchSuffix
+	devel:libpangocairo-1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpthread$secondaryArchSuffix
+	devel:librt$secondaryArchSuffix
```

Comment:
No need for it, there is no librt

Reply:
It's overall to many here and in lib:* you could remove them (or comment them at the beginning of the line with #) and see what cmake is checking for ..